source/utilities.tex

```diff
@@ -16245,7 +16245,7 @@
 
 \pnum
 \throws
-Shall not throw exceptions when \tcode{f} is a function pointer
+Does not throw exceptions when \tcode{f} is a function pointer
```

Comment:
Same here, "_Throws:_ Nothing if ..."

Reply:
Updated. And caught some more instances of "shall not participate unless X" that can by the same logic be simplified to "participates only if X".

Reply:
> And caught some more instances of "shall not participate unless X" that can by the same logic be simplified to "participates only if X".

Please don't change that. The current phrasing is a consistent form that is used in hundreds of places, please don't change two of them. All occurrences will get replaced with _Constraints:_ elements by papers currently being reviewed by LWG. Please don't complicate that by making piecemeal editorial changes to them.

Reply:
Okay, updated again.